.github/workflows/buildtest.yml

```diff
@@ -61,7 +61,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        distro: ["ubuntu-18.04", "debian-10", "debian-9", "ubuntu-20.04"]
+        distro: ["ubuntu-18.04", "debian-11", "debian-10", "ubuntu-20.04"]
```

Comment:
I found the ordering somewhat confusing ;-)

Reply:
Yes, I fixed it :)